uptime.go

```diff
@@ -0,0 +1,324 @@
+package godo
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"path"
+)
+
+const uptimeChecksBasePath = "/v2/uptime/checks"
+
+// UptimeChecksService is an interface for creating and managing Uptime checks with the DigitalOcean API.
+// See: https://docs.digitalocean.com/reference/api/api-reference/#tag/Projects
+type UptimeChecksService interface {
+	List(context.Context, *ListOptions) ([]UptimeCheck, *Response, error)
+	Get(context.Context, string) (*UptimeCheck, *Response, error)
+	GetState(context.Context, string) (*UptimeCheck, *Response, error)
+	Create(context.Context, *CreateUptimeCheckRequest) (*UptimeCheck, *Response, error)
+	Update(context.Context, string, *UpdateUptimeCheckRequest) (*UptimeCheck, *Response, error)
+	Delete(context.Context, string) (*Response, error)
+	GetAlert(context.Context, string, string) (*Alert, *Response, error)
+	ListAlerts(context.Context, *ListOptions, string) ([]Alert, *Response, error)
```

Comment:
```suggestion
	ListAlerts(context.Context, string, *ListOptions) ([]Alert, *Response, error)
```

For consistency with other resources, `*ListOptions` should be the last argument. A few examples off the top of my head:

https://github.com/digitalocean/godo/blob/c871e00ee469869cc49d3c7ef90828b38ad903d1/kubernetes.go#L43

https://github.com/digitalocean/godo/blob/c871e00ee469869cc49d3c7ef90828b38ad903d1/droplets.go#L28

Reply:
ah, yes. same goes with `CreateAlert`, will update all.